### ISSUE TITLE
2.7.5 patch release plan (security release)

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2753]
- https://github.com/az-digital/az_quickstart/pull/2751

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick fc88c4c8b081e046dbea399905290ae03dfe9787
```

## Release notes snippet
`git log 2.7.4..HEAD --pretty='format:- %s'`
```
- Closes #2750: Update Drupal core to 10.0.11 for SA-CORE-006 (critical security update). (#2751)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.5
